### PR TITLE
fix: add Dhananjayan's data

### PR DESCRIPTION
**Pull request by dhananjayandayalan**

### CODE REVIEW

File: public/data/dhananjayan.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "url": "https://github.com/dhananjayandayalan",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/dhananstonks",
+        "icon": "twitter"
+      }
+    ]
+  }
```
Comment (krishguptadev) [inline public/data/dhananjayan.json:17]:
```suggestion
}

```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Review by Pradumnasaraf — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by shreyaansjain06 — APPROVED** [review]

**Comment by dhananjayandayalan** [comment]
What's workflow awaiting approval? Why It takes time to merge my pull request? I really don't know whether I can ask this question or not.

**Comment by krishguptadev** [comment]
> What's workflow awaiting approval? Why It takes time to merge my pull request? I really don't know whether I can ask this question or not.

We need a actual maintainer to review your PR before it is merged 😊

**Comment by krishguptadev** [comment]
Cc @schmelto

**Comment by dhananjayandayalan** [comment]
> > What's workflow awaiting approval? Why It takes time to merge my pull request? I really don't know whether I can ask this question or not.
> 
> We need a actual maintainer to review your PR before it is merged 😊

Thanks for clarifying😊

**Pull request merged**